CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ### Added 
 
+- Add `SitterProfileViewModel` for `SitterProfileView` - [#164](https://github.com/ios-course/ironfoudation-team-project/pull/164)
```

Comment:
This is a bit confusing since we don't have the `SitterProfileView` yet.
```suggestion
- Add `SitterProfileViewModel` - [#164](https://github.com/ios-course/ironfoudation-team-project/pull/164)
```